src/makeRobot.js

```diff
@@ -38,7 +39,59 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
```

Comment:
make spaces between properties
```suggestion
    get info() {return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
    },
    
    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
```